goon/browserassets/js/browserOutput.js

```diff
@@ -166,30 +151,76 @@ function emojiparse(el) {
 	}
 }
 
-// Colorizes the highlight spans
-function setHighlightColor(match) {
```

Comment:
You broke this by smashing its function, this will except when the user tries to set their highlight color

Reply:
This function was never used, which is why I deleted it.
Changing color still works
![dreamseeker_2019-11-04_19-42-24](https://user-images.githubusercontent.com/7831163/68152262-3c388080-ff3b-11e9-9941-4f8018fc2370.png)

Reply:
that's probably a bug, i presume it was meant to retroactively recolor previous highlights.

Reply:
No, it's not.
Highlights are only applied on new messages. Have always been.
I think it's intentional, as going through all the logs would be fairly slow (and since you probably don't really care about highlights on the old messages).

Reply:
it isnt a thing that happens often enough for speed to matter much anyways.